system/infra-monitoring/vendor/atlas/templates/configmap.yaml

```diff
@@ -86,6 +86,20 @@ data:
               region: "{{ .Values.global.region }}"
               status: "active"
 
+            - custom_labels:
+                batch: "cisco-ios-xr_core-router"
+                job: "network/ssh"
+                credential: "default"
+                device: "cisco-ios-xr"
+              metrics_label: "corerouter"
+              target: 1
+              manufacturer: "cisco"
+              region: "{{ .Values.global.region }}"
+              role: "core-router"
+              tenant: "converged-cloud"
+              q: "rt"
```

Comment:
Is there a reason for this query string? Cause it letls our the routers on the old naming convention.

Reply:
This exporter meant to be targeting only the ASR9K routers that don't support SNMP OIDs. The networking team provided me with this filter in netbox which will target those routers. If the naming convention is changed then they will provide me with the new fiter.

Reply:
Field filters were changes as requested after the proper fields were added in netbox.